### PR TITLE
Fix: Prettier breaks code when inline number lists are used as arbitrary arguments [SCSS]

### PR DESCRIPTION
**Pull request by boyenn**
Fixes: https://github.com/prettier/prettier/issues/8565

- [x] I’ve added tests to confirm my change works.
- [x] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.

Not sure if this warrants a changelog ? Might just clutter it ?
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by boyenn** [comment]
@evilebottnawi I've noticed that sometimes you guys skip changelogs on really trivial things.

What are your thoughts on this PR ? I'm worried that adding a changelog here would also just be clutter?

File: src/language-css/parser-postcss.js [hunk 1]
```diff
@@ -32,6 +32,18 @@ function parseValueNodes(nodes) {
   for (let i = 0; i < nodes.length; ++i) {
     const node = nodes[i];
 
+    if (
+      node.type === "number" &&
+      node.unit === ".." &&
+      node.value[node.value.length - 1] === "."
```
Comment (evilebottnawi) [inline src/language-css/parser-postcss.js:38]: Will be great to add `isScssSyntax()` check here, because we don't want to parse `...` for less and css

Comment (boyenn) [reply src/language-css/parser-postcss.js:38]: Can't find `isScssSyntax` anywhere in the codebase, do you mean checking options.parser ? 
If so, I don't have access to it in `parseValueNodes`

**Comment by thorn0** [comment]
Yes, it's okay to skip changelogging such entries. Or you can add this PR to the changelog entry of #8567 if you feel they're related closely enough.

**Comment by boyenn** [comment]
@thorn0 Combined it with the previous one. Tried my best to make it as readable as possible, but I'm not sure if I'm deviating from the standard too much ?

**Comment by boyenn** [comment]
@thorn0 Combined it with the previous one. Tried my best to make it as readable as possible, but I'm not sure if I'm deviating from the standard too much ?

File: src/language-css/parser-postcss.js (hunk 1 above)
Comment (boyenn) [reply src/language-css/parser-postcss.js:38]: @evilebottnawi please advise, I had another look but I can't find any way to determine whether or not I'm in scss syntax this far into the callstack

**Comment by fisker** [comment]
```markdown
#### ONE TITLE ([#pr-1](pr-1) by [@author-1](@author-1), [#pr-2](pr-2) by [@author-2](@author-2), [#pr-n](pr-n) by [@author-n](@author-n))
```

**Comment by boyenn** [comment]
Seems I was pretty damn close.

I followed https://github.com/prettier/prettier/pull/8551/files as example, since that is also one with both PR's of same author.

**Comment by fisker** [comment]
Check these two:

- [1](https://prettier.io/blog/2020/03/21/2.0.0.html#typescript-38-7631httpsgithubcomprettierprettierpull7631-by-thorn0httpsgithubcomthorn0-7764httpsgithubcomprettierprettierpull7764-by-sosukesuzukihttpsgithubcomsosukesuzuki-7804httpsgithubcomprettierprettierpull7804-by-sosukesuzukihttpsgithubcomsosukesuzuki)
- [2](https://prettier.io/blog/2020/03/21/2.0.0.html#always-add-a-space-after-the-function-keyword-3903httpsgithubcomprettierprettierpull3903-by-j-f1httpsgithubcomj-f1-josephfrazierhttpsgithubcomjosephfrazier-sosukesuzukihttpsgithubcomsosukesuzuki-thorn0httpsgithubcomthorn0-7516httpsgithubcomprettierprettierpull7516-by-bakkothttpsgithubcombakkot)

Hah

File: src/language-css/parser-postcss.js (hunk 1 above)
Comment (evilebottnawi) [reply src/language-css/parser-postcss.js:38]: Just pass it to `parseNestedCSS` and other nested functions

Comment (boyenn) [reply src/language-css/parser-postcss.js:39]: Done!

File: changelog_unreleased/css/pr-8567.md [hunk 2]
```diff
@@ -20,3 +22,23 @@ body {
...
+
+/* Prettier stable */
+body {
+  background-color: rgba(50 50 50 50..);
+}
+
+/* Prettier master */
+body {
+  background-color: rgba(50 50 50 50...);
+}
```
Comment (fisker) [inline changelog_unreleased/css/pr-8567.md:43]: Can  we  use  just  one  code  block ?

**Pull request merged**